app-backend/datamodel/src/main/scala/Scope.scala

```diff
@@ -592,6 +600,21 @@ object Scopes {
           Action.AddScenes,
           Action.ReadPermissions
         ).map(makeScopedAction(Domain.AnnotationProjects, _, None)) ++
+          Set(
+            Action.Read,
+            Action.Update,
+            Action.Delete,
+            Action.CreateAnnotation,
+            Action.DeleteAnnotation,
+            Action.UpdateAnnotation,
```

Comment:
I don't this this is used -- as with the ones below, the actions that aren't used shouldn't be included in the scope. I didn't check all of them, but this `Set` should be shrunk to the ones that are actually checked in routes